kolibri/core/assets/src/views/auth-message.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+
+  <div class="auth-message">
+    <h1>
+      {{ header || $tr('loginPrompt') }}
```

Comment:
Can simplify by making ` $tr('loginPrompt') ` the default value for the `header` prop?